### ISSUE TITLE
in 1.11.0 sslmode = "disable" does not work properly

### ISSUE THREAD

**Issue by ZoraBoris** [cyrilgdn/terraform-provider-postgresql#36]
I have used latest version 1.11.0 and have issues with 
`pq: SSL is not enabled on the server`
I have downgraded to 1.9.0 and everything work fine

context:

```hcl
terraform {
  required_providers {
    kubernetes = {
      source = "hashicorp/kubernetes"
      version = "1.13.3"
    }
    postgresql = {
      source = "cyrilgdn/postgresql"
      version = "1.9.0"
    }
  }
}

provider "kubernetes" {}

// kubectl port-forward postgres-6b66657994-j2bks 25432:5432
provider "postgresql" {
  host = "127.0.0.1"
  port = "25432"
  database = "postgres"
  username = ******
  password = ******
  sslmode = "disable"
}
```

**Comment by izinovik** [comment]
Having the same issue. Switched back to 1.10.0 which works fine.

**Comment by rtrive** [comment]
i can confirm, with 1.11.0 i give the SSL error but with 1.10.0 no

**Comment by mcwarman** [comment]
As a workaround setting the environment variable still seems to work:

```
export PGSSLMODE=disable
```

**Comment by cyrilgdn** [comment]
Fixed in  [v1.11.1](https://registry.terraform.io/providers/cyrilgdn/postgresql/latest)
